crypto/noise/Cargo.toml

```
@@ -19,7 +19,7 @@ secrecy = "0.8"
 serde = { version = "1.0", default-features = false, features = ["alloc", "derive"] }
 sha2 = { version = "0.10", default-features = false }
 subtle = { version = "2.4.1", default-features = false, features = ["i128"] }
-zeroize = "1.5.4"
+zeroize = "1.5.5"
```

Optional: Specify only the major and minor versions, so future updates don't require modifying `Cargo.toml`, only `Cargo.lock`
```suggestion
zeroize = "1.5"
```